src/miral/window_management_trace.cpp

```diff
@@ -138,28 +140,42 @@ auto dump_of(miral::WindowInfo const& info) -> std::string
         APPEND(state);
         bout.append("top_left", info.window().top_left());
         bout.append("size", info.window().size());
-        if (info.state() != mir_window_state_restored) APPEND(restore_rect);
+        if (info.state() != mir_window_state_restored)
+            APPEND(restore_rect);
         if (std::shared_ptr<mir::scene::Surface> parent = info.parent())
             bout.append("parent", parent->name());
         bout.append("children", dump_of(info.children()));
-        if (info.min_width()  != miral::default_min_width)  APPEND(min_width);
-        if (info.min_height() != miral::default_min_height) APPEND(min_height);
-        if (info.max_width()  != miral::default_max_width)  APPEND(max_width);
-        if (info.max_height() != miral::default_max_height) APPEND(max_height);
-        if (info.width_inc()  != miral::default_width_inc)  APPEND(width_inc);
-        if (info.height_inc() != miral::default_height_inc) APPEND(height_inc);
-        if (info.min_aspect() != miral::default_min_aspect_ratio) APPEND(min_aspect);
-        if (info.max_aspect() != miral::default_max_aspect_ratio) APPEND(max_aspect);
-        if (info.depth_layer() != mir_depth_layer_application) APPEND(depth_layer);
-        if (info.attached_edges() != 0) APPEND(attached_edges);
-        if (info.exclusive_rect() != mir::geometry::Rectangle{{}, {}}) APPEND(exclusive_rect);
+        if (info.min_width() != miral::default_min_width)
+            APPEND(min_width);
+        if (info.min_height() != miral::default_min_height)
+            APPEND(min_height);
+        if (info.max_width() != miral::default_max_width)
+            APPEND(max_width);
+        if (info.max_height() != miral::default_max_height)
+            APPEND(max_height);
+        if (info.width_inc() != miral::default_width_inc)
+            APPEND(width_inc);
+        if (info.height_inc() != miral::default_height_inc)
+            APPEND(height_inc);
+        if (info.min_aspect() != miral::default_min_aspect_ratio)
+            APPEND(min_aspect);
+        if (info.max_aspect() != miral::default_max_aspect_ratio)
+            APPEND(max_aspect);
+        if (info.depth_layer() != mir_depth_layer_application)
+            APPEND(depth_layer);
+        if (info.attached_edges() != 0)
+            APPEND(attached_edges);
+        if (info.exclusive_rect() != mir::geometry::Rectangle{{}, {}})
+            APPEND(exclusive_rect);
```

Comment:
I know the "pre" of this block is an exception to our usual style, but I think it is easier to maintain than the "post".

OTOTH it is possible to disable clang-format for a block so exceptions can be made:
```c++
// clang-format off
    void    unformatted_code  ;
// clang-format on
```